.evergreen/config.yml

```diff
@@ -542,6 +576,19 @@ post:
 ignore:
   - '*.md'
 tasks:
+  - name: test-load-balancer
```

Comment:
Testing load balancer mode starts HAProxy, starts a sharded cluster, runs the lb tests, then stops HAProxy